back/spec/factories/ideas.rb

```diff
@@ -44,6 +44,7 @@
     factory :native_survey_response, class: 'Idea' do
       association :project, factory: :single_phase_native_survey_project
       creation_phase { project.phases.first }
+      idea_status { nil }
```

Comment:
This is the change that makes the specs more consistent with the web application.